spacy/tests/test_cli.py

```diff
@@ -16,7 +16,10 @@
 from spacy.cli.debug_data import _get_labels_from_spancat
 from spacy.cli.download import get_compatibility, get_version
 from spacy.cli.init_config import RECOMMENDATIONS, init_config, fill_config
-from spacy.cli.package import get_third_party_dependencies
+from spacy.cli.package import (
+    get_third_party_dependencies,
+    _is_permitted_package_name,
+)
```

Comment:
We avoid the multiline imports in our code:

```suggestion
from spacy.cli.package import get_third_party_dependencies
from spacy.cli.package import _is_permitted_package_name
```